requirements/static/py3.7/windows.txt

```diff
@@ -108,7 +108,7 @@ smmap==0.9.0
 strict-rfc3339==0.7
 tempora==1.14.1           # via portend
 timelib==0.2.4
-tornado==4.5.3 ; python_version >= "3.4"
+tornado==4.5.3
```

Comment:
shouldn't tornado be removed in these windows.txt files and the darwin.txt files since tornado was removed from their `*.in` files?

Reply:
This is auto generated, so something weird happened. I'll look into it.